Evolve.Migrations.Helper.Tests/OutputHelper.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.IO;
+using System.Text;
+
+namespace Evolve.Migrations.Helper.Tests
+{
+    public static class OutputHelper
+    {
+        public static string ExecuteAndReadOutputText(Action action)
+        {
+            using var stream = new MemoryStream();
+            
+            var writer = new StreamWriter(stream);
+            Console.SetOut(writer);
+            action();
+            Console.Out.Flush();
+
+            var buffer = new byte[stream.Length];
+            stream.Seek(0, SeekOrigin.Begin);
+            var _ = stream.Read(buffer, 0, buffer.Length);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable '_'.](https://app.codacy.com/manual/odair-pedro/evolve-migrations-helper/pullRequest?prid=5663793)